src/components/button.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {FC} from 'react';
+import {Text, Dimensions, StyleSheet, View} from 'react-native';
+import {TextInput, TouchableOpacity} from 'react-native-gesture-handler';
+const {height, width} = Dimensions.get('screen');
+
+interface Props {
+  title: string;
+  onSubmit: () => void;
+}
+
+const Button: FC<Props> = ({title, onSubmit}) => {
+  return (
+    <>
+      <TouchableOpacity style={styles.container} onPress={onSubmit}>
+        <Text style={styles.text}>{title}</Text>
+      </TouchableOpacity>
+    </>
+  );
+};
+
+export default Button;
+
+const styles = StyleSheet.create({
+  container: {
+    alignSelf: 'center',
+    backgroundColor: '#4267B2',
```

Comment:
for now, this is fine....but for projects .. we should have global themes .. from where we should import colors, fonts

Reply:
Okay Got it